main.go

```diff
@@ -117,7 +117,7 @@ func init() {
 func CommandFunc(cmd *cobra.Command, args []string) error {
 	var rs string
 	if len(disclaimer) > 0 {
-		rs += disclaimer + "\n\n\n"
+		rs += disclaimer + "\n\n"
```

Comment:
Can we trim space here, to ensure that if the user passes a linebreak at the end, the Markdown is still valid?

```suggestion
		rs += strings.TrimSpace(disclaimer) + "\n\n"
```

Reply:
@ivanvc Thanks for the suggestion. That was a good catch. Have made the necessary changes.